### ISSUE TITLE
Wrong response of a checkbox field in a form

### ISSUE THREAD

**Issue by rolfn** [vitmalina/w2ui#866]
In the following small example I expect different response values if I change the checkbox but I get always the string "on". No matter what alert call is used.
``` html
<html>
<head>
  <script type="text/javascript"
    src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js">
  </script>
  <link rel="stylesheet" type="text/css" href="http://w2ui.com/src/w2ui-1.4.min.css" />
  <script type="text/javascript" src="http://w2ui.com/src/w2ui-1.4.min.js"></script>
  </script>
</head>
<body>

  <div id="myForm">
    <div class="w2ui-field">
      <div><input name="myCheckbox" type="checkbox" /></div>
    </div>
  </div>

<script type='text/javascript'>
  $('#myForm').w2form({
    name   : 'myForm',
    fields : [{ name: 'myCheckbox', type: 'checkbox' }],
  });
  w2ui.myForm.onChange = function (target, data) {
    alert(target + ':' + data.value_new);
    data.onComplete = function(d) {
      //alert(target + ':' + d.value_new);
    };
  };
</script>
</body>
</html>
```
Thanks in advance.
...Rolf

**Comment by vitmalina** [comment]
Please create a js fiddle out of it.

**Comment by rolfn** [comment]
I tried it:
https://jsfiddle.net/3ohmmmce/5/
Unfortunately the fiddle does not work. Maybe you can make it work to see the bug? Thanks.

...Rolf

**Comment by vitmalina** [comment]
I fixed the issue. Sorry it took me so long. Now in the event object value_new will be correct. I have fixed in both 1.4 branch and current master. However, there is a way to make it work with old code. Here is JS fiddle: http://jsfiddle.net/3ohmmmce/7/

Here is fixed 1.4.x version that will be 1.4.3 https://raw.githubusercontent.com/vitmalina/w2ui/master/dist/w2ui.js

**Comment by rolfn** [comment]
Thank you.
